### DISCUSSION TITLE
Duplicates should be editable and replaced with <use>

### DISCUSSION THREAD

**Discussion by madsenfr** [SVG-Edit/svgedit#805] [New feature request]
## Motivation

When I duplicate a shape several times, and then want to change this shape everywhere, I would rather edit code and change duplicate by using tag `<use href="#myId" .../>`.

See details here : https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use

## Current behavior

`<use/>` is deleted after editing code.

## Desired behavior

`<use/>` should be kept in the code and interpreted.

## Alternatives considered

No alternative found, except by duplicating shapes which become independant shapes.

**Comment by jfhenon** [comment]
Your suggestion makes sense but it need to be an additional behavior as some users will likely ask to keep the current behavior.
We would need a PR on V7 to get this enhancement.
